Services/Administration/OJS.Services.Administration.Business/Implementations/ContestCategoriesCacheService.cs

```diff
@@ -73,8 +112,13 @@ private async Task RemoveCacheFromCategory(int contestCategoryId)
             CacheConstants.ContestParentCategoriesFormat,
             contestCategoryId);
 
-        await this.cache.Remove(categoryNameCacheId);
-        await this.cache.Remove(subCategoriesCacheId);
-        await this.cache.Remove(parentCategoriesCacheId);
+        await this.RemoveCaches([
+            categoryNameCacheId,
+            subCategoriesCacheId,
+            parentCategoriesCacheId,
```

Comment:
`categoryName` and `parentCategoriesCacheId` are not used anywhere. We should remove those keys.

Reply:
Done.